habitat-lab/habitat/datasets/pointnav/pointnav_dataset.py

```diff
@@ -123,6 +121,8 @@ def __init__(self, config: Optional["DictConfig"] = None) -> None:
                     self.from_json(f.read(), scenes_dir=config.scenes_dir)
 
         else:
+            with gzip.open(datasetfile_path, "rt") as f:
```

Comment:
Hmm so no one was using `has_individual_scene_files` earlier?

Reply:
I think other datasets did have the episode split, like HM3D.